source/common/config/grpc_mux_impl.h

```diff
@@ -42,7 +43,9 @@ class GrpcMuxImpl : public GrpcMux,
   void onRemoteClose(Grpc::Status::GrpcStatus status, const std::string& message) override;
 
   // TODO(htuch): Make this configurable or some static.
-  const uint32_t RETRY_DELAY_MS = 5000;
+  const uint32_t RETRY_INITIAL_DELAY_MS = 5000;
```

Comment:
Now that we have sensible backoff, I think you should shrink this down to something very small for the initial retry, e.g. 500ms. This will fix the problem that Istio are seeing (@costinm) in some cases where Pilot isn't ready. The exponential backoff will take us to a large number pretty quickly.

Reply:
Yes. I will change